howfairis/cli.py

```diff
@@ -94,26 +100,53 @@ def cli(url=None, branch=None, config_file=None, remote_config_file=None, path=N
         print("It seems you have not yet added the fair-software.eu badge to " +
               "your {0}. You can do so by pasting the following snippet:\n\n{1}"
               .format(checker.readme.filename, badge))
+        github_readme_creation_check(url, checker.readme.filename, checker.repo.platform, branch)
         sys.exit(1)
 
     if current_compliance == previous_compliance:
         print("Expected badge is equal to the actual badge. It's all good.\n")
+        github_readme_creation_check(url, checker.readme.filename, checker.repo.platform, branch)
         sys.exit(0)
 
     if current_compliance > previous_compliance:
         print("Congratulations! The compliance of your repository exceeds " +
               "the current fair-software.eu badge in your " +
               "{0}. You can replace it with the following snippet:\n\n{1}"
               .format(checker.readme.filename, badge))
+        github_readme_creation_check(url, checker.readme.filename, checker.repo.platform, branch)
         sys.exit(1)
 
     print("The compliance of your repository is different from the current " +
           "fair-software.eu badge in your " +
           "{0}. Please replace it with the following snippet:\n\n{1}"
           .format(checker.readme.filename, badge))
-
+    github_readme_creation_check(url, checker.readme.filename, checker.repo.platform, branch)
     sys.exit(1)
 
 
+def github_readme_creation_check(url, filename, platform, branch):
+    if platform != Platform.GITHUB:
+        return(0)
```

Comment:
Do you intend to use the int 0? If you do, `return 0` is the correct style

Reply:
Stefan pointed out that we should probably raise or warn depending on whether we have already implemented something. E.g. `raise NotImplementedException`

Reply:
Ok, the numbers have been removed from the return statements. Also this if statement has been moved up one layer to make clear that the check should only be performed for Github repos